frepple/controllers/outbound.py

```diff
@@ -1448,63 +1457,73 @@ def export_purchaseorders(self):
             "order_id",
             "state",
         ]
-        po_line = [i for i in recs.read(fields)]
+        po_lines = [i for i in all_po_lines.read(fields)]
 
         # Get all purchase orders
-        m = self.env["purchase.order"]
-        ids = [i["order_id"][0] for i in po_line]
-        fields = ["name", "company_id", "partner_id", "state", "date_order"]
-        po = {}
-        for i in m.browse(ids).read(fields):
-            po[i["id"]] = i
+        all_po_ids = [po_line["order_id"][0] for po_line in po_lines]
+        fields = [
+            "name",
+            "company_id",
+            "partner_id",
+            "state",
+            "date_order"
+        ]
+
+        purchase_orders = {}
+        for purchase_order in purchaseOrder.browse(all_po_ids).read(fields):
+            purchase_orders[purchase_order["id"]] = purchase_order
 
         # Create purchasing operations
         yield "<!-- open purchase orders -->\n"
         yield "<operationplans>\n"
-        for i in po_line:
-            if not i["product_id"] or i["state"] == "cancel":
+        for po_line in po_lines:
+            if not po_line["product_id"] or po_line["state"] == "cancel":
                 continue
-            item = self.product_product.get(i["product_id"][0], None)
-            j = po[i["order_id"][0]]
+            po_line_product = self.product_product.get(po_line["product_id"][0], None)
+            purchase_order = purchase_orders[po_line["order_id"][0]]
             # if PO status is done, we should ignore this PO line
-            if j["state"] == "done":
+            if purchase_order["state"] == PurchaseOrderStates.DONE:
                 continue
             location = self.mfg_location
-            if location and item and i["product_qty"] > i["qty_received"]:
+            if location and po_line_product and po_line["product_qty"] > po_line["qty_received"]:
+                if purchase_order == PurchaseOrderStates.RFQ:
+                    start = (
```

Comment:
Deze berekening wordt in feite genegeerd doordat de start en end variabelen verderop worden herberekend in lijnen 1507 en 1512.
Doet geen kwaad maar was misschien niet de bedoeling?

Reply:
Dag Johan,

Ik was blijkbaar die oude start en end datums eruit te halen...
Bij deze heb ik ze eruit gehaald en doet de nieuwe code ook iets haha